setup.py

```diff
@@ -18,7 +18,6 @@ def read(fname):
     description="SQLAlchemy to Django integration library",
     long_description=read("README.rst"),
     packages=find_packages(),
-    install_requires=["sqlalchemy>0.7.1"],
     classifiers=[
         "Intended Audience :: Developers",
         "License :: OSI Approved :: BSD License",
```

Comment:
I think we don't need the setup.py if we are using poetry, right?

Reply:
I will try